orchard-ws/app/services/Metric.scala

```diff
@@ -60,6 +60,9 @@ trait Metric {
 class PrometheusMetric @Inject() (implicit ec: ExecutionContext) extends Metric {
 
   DefaultExports.initialize()
+  // clear the default registry uppon initialization to make it easier to work with play-plugin
+  // auto recompile
+  clear()
```

Comment:
I am thinking would it be better to call 
`CollectorRegistry.defaultRegistry.clear()` here instead of `clear()`
because in this context `clear()` could be interpreted as to clear the values in the collector rather than the registry.  

And leave `clear()` empty for possible extension?

Reply:
I like Simon's idea.  And I like that the tests now pass without needing the `afterEach`.

Reply:
what's the difference between clear the values in the collector vs the registry?

Reply:
The dependency `CollectorRegistry` has 2 maps `collectorsToNames` and `namesToCollectors`.  The registry clear those 2 maps, forgetting all the collectors.  

`Collector` has a subtype `SimpleCollector` that has a `clear()` method operating on the children time series.  `Counter` and `Summary` extends `SimpleCollector`.